shotover-proxy/src/transforms/cassandra/sink_cluster/topology.rs

```diff
@@ -106,9 +124,21 @@ async fn topology_task_process(
                                     }
 
                                     nodes = new_nodes;
+
+                                    if let Err(watch::error::SendError(_)) =
+                                        nodes_tx.send(nodes.clone())
+                                    {
+                                        break 'listen;
```

Comment:
Can we just immediately `return Ok(())` instead of breaking out and then relying on the return that occurs later on?